charts/prow/templates/prow_configs.yaml

```diff
@@ -0,0 +1,64 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ template "fullname" . }}
+data:
+  config.yaml: |-
+    ---
+    plank:
+      job_url_template: '{{ template "job_url_template" . }}'
+      report_template: '{{ template "report_template" . }}'
+      
+    triggers:
+    - repos:
+      {{ range .Values.repos }}
+      - {{ . }}
+      {{ end }}
+      trusted_org: {{ .Values.trustedOrg }}
+      
+    presubmits:
+      
+      {{ range .Values.repos }}
```

Comment:
Yeah for now I'm just generating presubmit entries per repo with all the plugins enabled.  There are many ways to rework this.